client/python/datajunction/models.py

```diff
@@ -42,6 +42,14 @@ class NodeStatus(str, enum.Enum):
     INVALID = "invalid"
 
 
+class NodeType(str, enum.Enum):
+    """
+    DJ node types
+    """
+
+    METRIC = "metric"
```

Comment:
Hmm, does this need all the other node types too?

Reply:
Yes, of course. I just missed to fill the rest... Will fix.